panels/dev-service/ha-panel-dev-service.html

```diff
@@ -1,28 +1,38 @@
-<link rel="import" href="../../bower_components/polymer/polymer.html">
```

Comment:
It's very annoying that you decided to change the quote style because it makes it very difficult to see what imports have changed